plugins/pencilblue/templates/admin/sites/site_form.html

```diff
@@ -0,0 +1,40 @@
+^tmp_admin=head^
+<div class="col-md-9">
+    ^tmp_admin=elements=error_success^
+    ^tmp_admin=elements=sub_nav^
+    ^tmp_admin=elements=tab_nav^
+    <form name="siteForm" novalidate>
+        <fieldset>
+            <div class="tab-content">
+                <div class="tab-pane active">
+                    <div class="form-group" ng-class="{'has-error': !isFieldValid(siteForm.displayName)}">
+                        <label>Display Name</label>
+                        <input type="text" name="displayName" class="form-control" ng-model="displayName" required>
+                        <div class="form_error" ng-if="!isFieldValid(siteForm.display)">^loc_REQUIRED_FIELD^</div>
+                    </div>
+                    <div class="form_group" ng-class="{'has-error': !isFieldValid(siteForm.hostname)}">
+                        <label>Hostname</label>
+                        <input type="text" name="hostname" class="form-control" ng-disabled="isActive" ng-model="hostname" required>
+                        <div class="form_error" ng-if="!isActive && !isFieldValid(siteForm.hostname)">^loc_REQUIRED_FIELD^</div>
+                        <div class="form_error" ng-if="isActive">^loc_SITE_MUST_DEACTIVATE^</div>
+                    </div>
+                </div>
+                <br>
+                <a class="btn btn-default" href="/admin/sites">
+                    <i class="fa fa-ban"></i>&nbsp;^loc_CANCEL^
+                </a>
+                <a class="btn btn-primary" ng-click="newSite()" ng-if="isNew">
```

Comment:
We can use the save button template and route to the correct functionality in the Angular controller.